srcs/layouts/deva_alt.xml

```diff
@@ -3,28 +3,27 @@
   <row>
     <key shift="0.35" width="0.9" key0="क" key1="ख" key2="घ" key3="ङ"  key4="ग"/>
     <key              width="0.9" key0="च" key1="छ" key2="झ" key3="ञ"  key4="ज"/>
-    <key              width="0.9" key0="ट" key1="ठ" key2="ढ" key3="ण"  key4="ड"/>
-    <key              width="0.9" key0="त" key1="थ" key2="ध" key3="न"  key4="द"/>
-    <key              width="0.9" key0="प" key1="फ" key2="भ" key3="म"  key4="ब"/>
+    <key              width="0.9" key0="ट" key1="ठ" key2="ढ" key3="ण"  key4="ड" anticircle="७"/>
```

Comment:
The anticircle might be hard to discover. Perhaps you can add `indication="७"` on every keys with a gesture to make them visible to users ?